ui/app/components/auth/form-template.ts

```diff
@@ -77,8 +66,36 @@ export default class AuthFormTemplate extends Component<Args> {
 
   constructor(owner: unknown, args: Args) {
     super(owner, args);
-    this.selectedAuthMethod = this.args.preselectedAuthType || 'token';
-    this.fetchMounts.perform();
+    // ensures args have settled before setting form state
+    setTimeout(() => this.initializeState(), 0);
+  }
+
+  initializeState() {
+    // FORMAT MOUNT DATA
+    this.authTabData = this._formatTabs();
```

Comment:
formatting happens here instead of in the route because there will be some additional logic if the cluster has login customizations (like a default method set)